mac/KeymanEngine4Mac/KeymanEngine4MacTests/PlatformTest.kmn

```diff
@@ -0,0 +1,80 @@
+﻿store(&TARGETS) 'any'
+store(&NAME) 'PlatformTest'
+begin Unicode > use(main)
+
+group(main) using keys 
+
+platform('touch') + [K_P] > ' Touch'
+platform('hardware') + [K_P] > ' Hardware'
++ [K_P] > ' [UI Undefined]'
+
+platform('TOUCH') + [SHIFT K_P] > ' Touch'
+platform('HARDWARE') + [SHIFT K_P] > ' Hardware'
++ [SHIFT K_P] > ' [UI Undefined]'
+
+platform('windows') + [K_O] > ' Windows'
+platform('android') + [K_O] > ' Android'
+platform('iOS') + [K_O] > ' iOS'
+platform('linux') + [K_O] > ' Linux'
+platform('macosx') + [K_O] > ' macOS'
++ [K_O] > ' [OS Undefined]'
+
+platform('WINDOWS') + [SHIFT K_O] > ' Windows'
+platform('ANDROID') + [SHIFT K_O] > ' Android'
+platform('IOS') + [SHIFT K_O] > ' iOS'
+platform('LINUX') + [SHIFT K_O] > ' Linux'
+platform('MacOSX') + [SHIFT K_O] > ' macOS'
++ [SHIFT K_O] > ' [OS Undefined]'
+
+platform('mac') + [K_L] > ' mac'
++ [K_L] > ' [Not a mac]'
+platform('MAC') + [SHIFT K_L] > ' mac'

```

Comment:
Can you have the case-insensitive variations of `mac`, `macOS`, and `macOSX` output a consistent `macOS`?